src/libtiled/imagelayer.cpp

```diff
@@ -84,7 +84,7 @@ bool ImageLayer::loadFromImage(const QImage &image, const QString &source)
 
 bool ImageLayer::loadFromImage(const QUrl &url)
 {
-    return loadFromImage(ImageCache::loadImage(url.toLocalFile()), url);
+    return loadFromImage(ImageCache::loadImage(Tiled::urlToLocalFileOrQrc(url)), url);
```

Comment:
```suggestion
    return loadFromImage(ImageCache::loadImage(urlToLocalFileOrQrc(url)), url);
```
Just suggesting to remove the namespace, because it's superfluous.

Reply:
Never mind, I see all the other invokations also qualify the namespace, so let's keep it for consistency.